### ISSUE TITLE
SBFspot Installation on Raspberry PI V3

### ISSUE THREAD

**Issue by SBFspot** [SBFspot/SBFspot#153]
Hello, as Raspberry PI V3 and new the Rasbian OS supports bluetooth already, I have not installed a new bluetooth driver (to avoid trouble), but at the end  I got an compile error, caused by "bluetooth/bluetooth.h: No such file or directory".

What should or can I do to solve it?

#### This work item was migrated from CodePlex
CodePlex work item ID: '149'
Vote count: '1'

**Comment by SBFspot** [comment]
[ortmueller@18/03/2016]
Hello,

what kind of Image did you use?
I used jessie light from here:
https://www.raspberrypi.org/downloads/raspbian/

Did you install bluetooth and libluetooth-dev decribed in the documentation?
libboost-all-dev also recommented.
Did you test hcitool scan with some results? - No driver needed.

Good luck

Timo

**Comment by SBFspot** [comment]
[user4711@19/03/2016]
It is running now.

I installed the Bluetooth driver as requested in the installation manual and then it works.

The hcitool works with and without installing the Bluetooth driver on Raspberry PI 3.

I have the newest version of Rasbian OS, Feb 2016.  On a Raspberry PI 3 Bluetooth is already supported, but not things like Bluetooth header files. Therefore I got an compile error

Thanks for your help.

user4711
